File: src/pocketcloud/command/impl/general/ReloadCommand.php
<?php

namespace pocketcloud\command\impl\general;

use pocketcloud\command\Command;
use pocketcloud\command\sender\ICommandSender;
use pocketcloud\PocketCloud;

class ReloadCommand extends Command {

    public function execute(ICommandSender $sender, string $label, array $args): bool {
        if (!PocketCloud::getInstance()->isReloading()) {
            PocketCloud::getInstance()->reload();
        }
        return true;
    }
}

File: src/main/java/de/pocketcloud/cloud/console/command/impl/ReloadCommand.java
package de.pocketcloud.cloud.console.command.impl;

import de.pocketcloud.cloud.PocketCloud;
import de.pocketcloud.cloud.console.command.Command;
import de.pocketcloud.cloud.console.command.ctx.CommandContext;
import de.pocketcloud.cloud.console.command.desc.CommandDescription;
import de.pocketcloud.cloud.console.command.sender.CommandSender;

@CommandDescription(name = "reload", description = "Reload the cloud", aliases = {"rl"})
public final class ReloadCommand extends Command {

    @Override
    public void prepare() {}

    @Override
    public boolean run(CommandSender sender, CommandContext ctx) {
        PocketCloud.instance().reload();
        return true;
    }
}